mongoexport/options.go

```diff
@@ -33,7 +33,7 @@ type OutputFormatOptions struct {
 	Type string `long:"type" value-name:"<type>" default:"json" default-mask:"-" description:"the output format, either json or csv (defaults to 'json')"`
 
 	// Deprecated: allow legacy --csv option in place of --type=csv
-	CSVOutputType bool `long:"csv" default:"false" hidden:"true"`
```

Comment:
With the newer version go-flags, the default for boolean doesn't work. Showing errors 
```
boolean flag `csv' may not have default values, they always default to `false' and can only be turned on
```
So I have to remove all the false boolean default value for flags.